simulation/environment/GIS.py

```diff
@@ -88,18 +84,9 @@ def __init__(self, api_key, origin_coord, dest_coord, waypoints, race_type, gola
         if api_call_required or force_update:
             logging.warning("New route requested and/or route save file does not exist. "
                             "Calling Google API and creating new route save file...\n")
-            logging.warning(
-                "The GIS class is collecting data from a Google API. Set force_update to false to prevent this and "
-                "read data from disk instead. This should improve performance. \n")
-            self.path = self.update_path(self.origin_coord, self.dest_coord, self.waypoints)
-            self.path_elevations = self.calculate_path_elevations(self.path)
-            self.path_time_zones = self.calculate_time_zones(self.path)
-            self.launch_point = self.path[0]
+            logging.error("Update API cache by calling CacheAPI.py , Exiting simulation...\n")
```

Comment:
Remove the `force_update` flag from `GIS`, `WeatherForecast`, and `config` as it now does nothing